Code/db/albums/init.sql

```diff
@@ -0,0 +1,14 @@
+CREATE DATABASE IF NOT EXISTS albums_db;
+USE albums_db;
+
+-- Add table creation scripts here
+
+CREATE TABLE albums (
+    album_id INT AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(100) NOT NULL,
+    release_date DATE NOT NULL,
+    band_name VARCHAR(100) NOT NULL,
```

Comment:
use band_id instead of band_name, as we will use the band_id to query the band and band_name could change